test/test_api/test_api.py

```diff
@@ -932,6 +932,63 @@ def test_pipeline_fit(openml_id,
             assert not os.path.exists(cv_model_path)
 
 
+@pytest.mark.parametrize('openml_id', (40984,))
+@pytest.mark.parametrize("budget", [1])
```

Comment:
These could be combined with: 

`@pytest.mark.parametrize('openml_id,budget', [(40984, 1)])`

Reply:
Okay Done.